pages/validators/how_to_set_up_full_node.md

```diff
@@ -4,60 +4,124 @@
 ```bash
-DYDX_HOME=/home/vmware/.dydx-mainnet-1
+sudo tee /etc/systemd/system/dydxprotocold.service > /dev/null << EOF
+[Unit]
+Description=osmosis node service
+After=network-online.target
+
+[Service]
+User=$USER
+ExecStart=/$HOME/go/bin/cosmovisor run start --non-validating-full-node=true
+WorkingDirectory=$HOME/.dydxprotocol
+Restart=always
+RestartSec=5
+LimitNOFILE=4096
+Environment="DAEMON_HOME=$HOME/.dydxprotocol"
+Environment="DAEMON_NAME=dydxprotocold"
+Environment="DAEMON_ALLOW_DOWNLOAD_BINARIES=false"
+Environment="DAEMON_RESTART_AFTER_UPGRADE=true"
+Environment="UNSAFE_SKIP_BACKUP=true"
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+sudo systemctl daemon-reload
+sudo systemctl enable dydxprotocold
 
-nohup dydxprotocold start --p2p.seeds="ade4d8bc8cbe014af6ebdf3cb7b1e9ad36f412c0@seeds.polkachu.com:23856,65b740ee326c9260c30af1f044e9cda63c73f7c1@seeds.kingnodes.net:23856,f04a77b92d0d86725cdb2d6b7a7eb0eda8c27089@dydx-mainnet-seed.bwarelabs.com:36656,20e1000e88125698264454a884812746c2eb4807@seeds.lavenderfive.com:23856,c2c2fcb5e6e4755e06b83b499aff93e97282f8e8@tenderseed.ccvalidators.com:26401,4f20c3e303c9515051b6276aeb89c0b88ee79f8f@seed.dydx.cros-nest.com:26656,a9cae4047d5c34772442322b10ef5600d8e54900@dydx-mainnet-seednode.allthatnode.com:26656,802607c6db8148b0c68c8a9ec1a86fd3ba606af6@64.227.38.88:26656,4c30c8a95e26b07b249813b677caab28bf0c54eb@rpc.dydx.nodestake.top:666,ebc272824924ea1a27ea3183dd0b9ba713494f83@dydx-mainnet-seed.autostake.com:27366" --home=$DYDX_HOME --non-validating-full-node=true > /tmp/fullnode.log 2>&1 &
 ```
-2. You can tail the log to see the progress.
+
+### Step 8: Get Snapshot
+> **Note:** the example values below align with the **deployment by DYDX token holders**. For alternatives, please visit the [Network Resources page](../network/resources.md).
+
+1. Choose a snapshot from https://docs.dydx.exchange/network/resources#snapshot-service
+2. Download and extract (using `lz4 -dc < snapshotfile.tar.lz4 | tar xf -`) the snapshot contents in the dydxprotocol home directory (make sure you are in this directory before running the tar command). In this example, the home directory is `$HOME/.dydxprotocol`
+
+### Step 9: Start Service
+
+```bash
+sudo systemctl start dydxprotocold
+```
+
+### Check logs for the service to ensure it is running
+
 ```bash
-tail -f /tmp/fullnode.log
+sudo journalctl -u dydxprotocold -f
 ```
-3. The full node is now syncing. To determine whether the full node is caught up with the chain head, please check the applicable block explorer to determine when it reaches the current block – an example block explorer is shown on https://www.mintscan.io/dydx
+The full node is now syncing. To determine whether the full node is caught up with the chain head, please check the applicable block explorer to determine when it reaches the current block – an example block explorer is shown on https://www.mintscan.io/dydx
```

Comment:
I would also add a small section on different interesting queries you can run. for ex, command to see how much the node has synced rather than parsing through logs